test/rsa_keygen_test.c

```diff
@@ -250,9 +250,18 @@ int main(int argc, char **argv)
 	} else {
 		printf("RSA keygen (%u) tests failed: %u errors.",
 		    key_bit_length, rc_test);
-		if (FIPS_mode())
+#ifdef ICA_FIPS
+		if (ica_fips_status()) {
 			printf(" (Parameters might be non FIPS conformant.)");
+#if OPENSSL_VERSION_NUMBER >= 0x30000000L
```

Comment:
are you using OPENSSL_VERSION_NUMBER because OPENSSL_VERSION_PREREQ is not available here?